react-hyf/react-hyf-client/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import HomeworkRouter from './Router/HomeworkRouter';
+import NotFound from './Router/NotFound';
+
+class App extends React.Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <Switch>
+          <Route exact path="/" component={HomeworkRouter} />
+          <Route path="/week1" component={HomeworkRouter} />
+          <Route path="/week2" component={HomeworkRouter} />
+          <Route path="/week3" component={HomeworkRouter} />
```

Comment:
You have same exact router you are routing to, perhaps having it as a component here would be more convenient.